### PR TITLE
ci(wordlist): Add acronyms and words to wordlist

### PR DESCRIPTION
**Pull request by peterjunpark**

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ AdaLoRA
 AddressSanitizer
 AlexNet
 Arb
+AutoAWQ
```
Comment (peterjunpark) [inline .wordlist.txt:23]: https://docs.vllm.ai/en/latest/quantization/auto_awq.html

File: .wordlist.txt [hunk 2]
```diff
@@ -87,6 +88,7 @@ ENDPGM
 EPEL
 EPYC
 ESXi
+EU
```
Comment (peterjunpark) [inline .wordlist.txt:91]: execution unit

File: .wordlist.txt [hunk 3]
```diff
@@ -187,6 +189,7 @@ MIGraphX
 MIOpen
 MIOpenGEMM
 MIVisionX
+MLIR
```
Comment (peterjunpark) [inline .wordlist.txt:193]: MLIR stands for Multi-Level Intermediate Representation

File: .wordlist.txt [hunk 4]
```diff
@@ -213,6 +216,7 @@ MyEnvironment
 MyST
 NBIO
 NBIOs
+NHWC
```
Comment (peterjunpark) [inline .wordlist.txt:220]: NHWC is a data layout format used in machine learning, particularly for tensors. It stands for N (batch size), H (height), W (width), and C (channels).

File: .wordlist.txt [hunk 5]
```diff
@@ -297,6 +301,7 @@ RelWithDebInfo
 Req
 Rickle
 RoCE
+Roofline
```
Comment (peterjunpark) [inline .wordlist.txt:305]: https://en.wikipedia.org/wiki/Roofline_model

File: .wordlist.txt [hunk 6]
```diff
@@ -347,6 +352,8 @@ TPU
 TPUs
 TRL
 TTFT
+TTGIR
```
Comment (peterjunpark) [inline .wordlist.txt:356]: In the context of Triton, which is an open-source machine learning compiler developed by OpenAI, TTGIR stands for Triton Tensor Graph Intermediate Representation

File: .wordlist.txt [hunk 7]
```diff
@@ -347,6 +352,8 @@ TPU
...
+TTIR
```
Comment (peterjunpark) [inline .wordlist.txt:357]: TTIR stands for Triton Tensor Intermediate Representation

File: .wordlist.txt [hunk 8]
```diff
@@ -377,6 +384,7 @@ VALU
 VBIOS
 VGPR
 VGPRs
+VGPU
```
Comment (peterjunpark) [inline .wordlist.txt:388]: virtual GPU

File: .wordlist.txt [hunk 9]
```diff
@@ -450,6 +458,7 @@ codename
 collater
 comgr
 completers
+composability
```
Comment (peterjunpark) [inline .wordlist.txt:462]: ability to compose

File: .wordlist.txt [hunk 10]
```diff
@@ -466,6 +475,7 @@ cuSOLVER
 cuSPARSE
 dataset
 datasets
+dataset's
```
Comment (peterjunpark) [inline .wordlist.txt:478]: possessive

File: .wordlist.txt [hunk 11]
```diff
@@ -583,6 +593,7 @@ optimizers
 os
 pageable
 parallelization
+parallelize
```
Comment (peterjunpark) [inline .wordlist.txt:597]: adapt (a program) for running on a parallel processing system.
"special software parallelizes C-based programs"

File: .wordlist.txt [hunk 12]
```diff
@@ -681,10 +694,12 @@ toolchain
 toolchains
 toolset
 toolsets
+torchtune
```
Comment (peterjunpark) [inline .wordlist.txt:698]: https://pytorch.org/blog/torchtune-fine-tune-llms/

File: .wordlist.txt [hunk 13]
```diff
@@ -681,10 +694,12 @@ toolchain
...
 torchvision
 tqdm
 tracebacks
 tunable
+tunings
```
Comment (peterjunpark) [inline .wordlist.txt:703]: As in "This is a tuning parameter; if you want the tunings to be chosen based on an average over multiple iterations, increase the allowed tuning duration."

File: .wordlist.txt [hunk 14]
```diff
@@ -670,7 +681,9 @@ subdirectory
 subexpression
 subfolder
 subfolders
+suboptimal
```
Comment (peterjunpark) [inline .wordlist.txt:685]: : less than optimal : not at the best possible level. suboptimal performance. suboptimal conditions/results/solutions.

File: .wordlist.txt [hunk 15]
```diff
@@ -670,7 +681,9 @@ subdirectory
...
 supercomputing
+templated
```
Comment (peterjunpark) [inline .wordlist.txt:687]: Simple past tense and past participle of template.

File: .wordlist.txt [hunk 16]
```diff
@@ -87,6 +88,8 @@ ENDPGM
 EPEL
 EPYC
 ESXi
+EU
+ExLlama
```
Comment (peterjunpark) [inline .wordlist.txt:92]: A standalone Python/C++/CUDA implementation of Llama for use with 4-bit GPTQ weights, designed to be fast and memory-efficient on modern GPUs.

**Review by samjwu — APPROVED** [review]

**Pull request merged**